lib/components/Button/button.scss

```diff
@@ -35,6 +35,7 @@
         display: inline-flex;
         align-items: center;
         justify-content: center;
+        outline-offset: 3px;
```

Comment:
Requested by designer. They need a offset for the outline on focus. We don't have 3px token, I added manually 3px here.